src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -32,6 +30,10 @@
     <PackageReference Include="ZeroFormatter.Analyzer" Version="1.1.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="..\..\harness\BenchmarkDotNet.Extensions\BenchmarkDotNet.Extensions.csproj" />
```

Comment:
Maybe the PATH could be something like `..\..\BenchmarkDotNet\Extensions\Extensions.csproj`?

Reply:
I am not sure if this is a better idea. I don't think that we are ever gonna need more `BenchmarkDotNet` projects but I think that we might add some other harnesses in the future.